packages/ui/color-picker/src/color-picker.tsx

```diff
@@ -47,16 +47,15 @@ const ColorPicker: React.FC<ColorPickerProps> = ({
 	// Update tempColor when color prop value changes
 	useEffect(() => {
 		setTempColor(color)
-
 		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [color])
+	}, [])
 
 	// Handle reset
 	const handleReset = useCallback(
 		(e) => {
 			e.stopPropagation()
-			setShowResetBtn(false)
 			setShowPicker(false)
+			setTempColor("")
```

Comment:
@pawan reset button should revert value to the default value and not clear it completely.

![image](https://github.com/wpmudev/sui-react/assets/14994452/8562850b-6470-456d-a7bd-530e8ae4499b)